python/dask_cudf/setup.py

```diff
@@ -10,8 +10,8 @@
 
 install_requires = [
     "cudf",
-    "dask>=2021.3.1",
-    "distributed>=2.22.0",
+    "dask==2021.3.1",
+    "distributed==2021.3.1",
```

Comment:
I don't think we want to change to an exact pin here, instead can we do `distributed>=2.22.0,<=2021.3.1`?